CK2Plus/decisions/settlement_decisions.txt

```diff
@@ -1358,7 +1347,7 @@ settlement_decisions = {
 
 			mult_modifier = {
 				factor = 100
-				FROM = { dynasty = 708 } # Ýrpád
+				FROM = { dynasty = 708 } # �rp�d
```

Comment:
This only displays wrong because GitHub uses UTF-8 encoding when displaying the file. I checked the file itself and it is correct.